File: src/main/java/br/com/correios/enderecador/cep/InterfaceEnderecoDao.java
package br.com.correios.enderecador.cep;

import br.com.correios.enderecador.bean.EnderecoBean;
import br.com.correios.enderecador.dao.DaoException;
import br.com.correios.enderecador.excecao.ConfiguracaoProxyException;

public interface InterfaceEnderecoDao {
    EnderecoBean consultar(String paramString, boolean paramBoolean) throws DaoException, ConfiguracaoProxyException;
}

File: src/main/kotlin/br/com/correios/enderecador/cep/InterfaceEnderecoDao.kt
package br.com.correios.enderecador.cep

import kotlin.Throws
import br.com.correios.enderecador.dao.DaoException
import br.com.correios.enderecador.excecao.ConfiguracaoProxyException
import br.com.correios.enderecador.bean.EnderecoBean

interface InterfaceEnderecoDao {
    @Throws(DaoException::class, ConfiguracaoProxyException::class)
    fun consultar(cep: String, proxy: Boolean): EnderecoBean?
}
